samples/getting-started/DemoApp/ViewController.swift

```diff
@@ -15,12 +15,31 @@ class ViewController: UIViewController {
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
-        label.text = "Hello World"
```

Comment:
please don't delete what we already had, but feel free to move the label somewhere else (outside the scroll view)

Reply:
(pasting in both relevant places) 
Very sorry about this, the changes to this file were not meant to be submitted. They were useful to me while developing, but my goal wasn't to change the DemoApp. I was pushing it to keep the git index clean, with the intention of resetting the file before re-requesting review.

Reply:
@janek I think it's great to add more functionality to the demo app, totally fine to leave it here, but please make the change additive rather than replacing what was there :)